frontend/src/metabase/dashboard/actions/data-fetching.js

```diff
@@ -207,6 +207,7 @@ export const fetchDashboard = createThunkAction(
 
       return {
         ...normalize(result, dashboard), // includes `result` and `entities`
+        dashboard: result,
```

Comment:
I pass `dashboard` so I don't need to use `Dashboards.HACK_getObjectFromAction(action)` in the reducer.